admin/javascript/dist/LeftAndMain.js

```diff
@@ -181,16 +181,13 @@
 				    basePath = getUrlPath($('base')[0].href);
 
 				if (basePath[basePath.length - 1] === '/') {
-					basePath += 'admin';
-				} else {
-					basePath = '/admin';
+					basePath = basePath.substring(0, basePath.length - 1);
 				}
-
 				_router2.default.base(basePath);
 
 				_config2.default.getTopLevelRoutes().forEach(function (route) {
-					(0, _router2.default)('/' + route + '/*', function (ctx, next) {
-						if (document.readyState !== 'complete' || typeof ctx.state.__forceReferer === 'undefined') {
```

Comment:
having the __forceReferer check here broke going "back" to the initial state, since it wasn't loaded via loadPanel().

Instead, sub-section loaders can conditionally avoid calling next() in their route handlers to prevent bubbling back up to this root controller.